pkg/network/setup/netpod/netpod_test.go

```diff
@@ -75,22 +77,49 @@ var _ = Describe("netpod", func() {
 		Expect(os.Setenv("MY_POD_IP", "10.10.10.10")).To(Succeed())
 	})
 
+	BeforeEach(func() {
+		cache := newConfigStateCacheStub()
+		state = netpod.NewState(cache, netnsStub{})
+	})
+
 	It("fails setup when reading nmstate status fails", func() {
 		netPod := netpod.NewNetPod(
-			nil, nil, vmiUID, 0, 0, 0,
+			[]v1.Network{*v1.DefaultPodNetwork()},
+			[]v1.Interface{{
+				Name:                   defaultPodNetworkName,
+				InterfaceBindingMethod: v1.InterfaceBindingMethod{Masquerade: &v1.InterfaceMasquerade{}},
+			}},
```

Comment:
Because this default interface is used in other tests with different bindings, I prefer to leave the binding part visible and be consistent through the test file.

Reply:
I think using the well know function reads better, it helps seeing the differences in tests that use interface with binding plugins.
Not a must though.